File: Lab2/Lab_2.cpp
/*Реализовать
представленные задачи на языках программирования C++, Python, Haskell
(только одно задание), C#, Go, Ruby (только одно задание), Java.

1. Дана сторона квадрата a. Найти квадрат его периметра P=4·a.
  С++:
*/
#include <iostream>
#include <math.h>

using namespace std;

int main() {

	int a,p,res;

	cin >> a;
	p = 4 * a;
	res = pow(p, 2);

	cout << res << endl;

	return 0;
}

File: Lab2/Lab_2.java
package Basics.Lessons;
import java.util.Scanner;

import static java.lang.Math.pow;

//ввод числа, *4, ^2
public class res {
public static void main(String[] args) {
  Scanner scanner = new Scanner(System.in);
  
int a = scanner.nextInt();
int b = (int) pow((a*4),2);

System.out.println(b);
  }
}
